### DISCUSSION TITLE
Debugging query don't work

### DISCUSSION THREAD

**Discussion by jhovannycanas** [adonisjs/core#3791] [Help]
Hello, 

I have configured the database query log and the listener as indicated in the following link and it does not work:
https://docs.adonisjs.com/guides/database/debugging#listening-to-the-event

In the start/events.ts file; I have configured other events, which do work,

**Comment by tomgobich** [comment]
If you're manually creating `start` directory files, make sure you're also adding them to the preloads array in your `.adonisrc.json` file otherwise they will not be executed when you boot your application.

If you use the Ace CLI to create your preload files, this will be taken care of for you.
```bash
node ace make:prldfile events
```

If the file is already in your preloads array, please share the code so we can help diagnose the issue.
